rest/server_context.go

```diff
@@ -35,7 +35,7 @@ const kStatsReportURL = "http://localhost:9999/stats"
 const kStatsReportInterval = time.Hour
 const kDefaultSlowQueryWarningThreshold = 500 // ms
 const KDefaultNumShards = 16
-const DefaultLogFreqencySecs = 5
+const DefaultLogFreqencySecs = 60
```

Comment:
I've already made this change on a local branch, just not pushed yet. It took me a while to figure out what this was when I stumbled upon it, and how it related to non-stats logging.

```suggestion
const DefaultStatsLogFreqencySecs = 60
```

Reply:
That's reasonable, and we might as well spell Frequency correctly while we're at it.